src/components/home/SideBarExternalNavLink.tsx

```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent } from "react";
+import { Stack } from "@mui/material";
+import theme from "theme";
+
+const activeBackground = "rgba(255, 255, 255, 0.1)";
```

Comment:
might be good to put this in the theme file so we can reuse it in other places 🤔

Reply:
👌 Adding to the theme.colors section

The nav link might need a typography specific section as well if we do reuse, or want a majority of styling to live in one place, but for now this seems like a unique item for the sidebar